static/nextant.sh

```diff
@@ -95,11 +108,13 @@ cd $NC_APPS_PATH
 tar zxf $NT_RELEASE
 bash $SCRIPTS/setup_secure_permissions_nextcloud.sh
 rm -r $NT_RELEASE
-sudo -u www-data php $NCPATH/occ app:enable nextant 
+sudo -u www-data php $NCPATH/occ app:enable nextant
+
 if [ $? -eq 0 ]
 then
     echo "Nextant app is now installed and enabled."
     echo "Please go to: Admin Settings --> Additional Settings, and configure the app"
+    echo -e "Finally run \033[0;32msudo -u www-data php /var/www/nextant/occ nextant:index \033[0mto get nextant running."
```

Comment:
The user can do that directly in the app settings (at the bottom) so it's necessary to instruct the user to go to admin settings the back to CLI when both can be done in app settings.